backend/src/test/java/com/festago/domain/TicketTest.java

```diff
@@ -20,96 +20,99 @@
 @SuppressWarnings("NonAsciiCharacters")
 class TicketTest {
 
-
     @Nested
     class 입장시간_추가_검증 {
 
         @ParameterizedTest
         @ValueSource(longs = {0, 1})
         void 입장시간이_티켓오픈시간_이전이면_예외(long minute) {
             // given
-            LocalDateTime now = LocalDateTime.now();
-            Stage stage = StageFixture.stage()
-                .startTime(now.plusDays(1))
-                .ticketOpenTime(now)
-                .build();
-            Ticket ticket = TicketFixture.ticket()
-                .stage(stage)
-                .build();
+            LocalDateTime stageStartTime = LocalDateTime.parse("2023-08-12T18:00:00");
+            LocalDateTime now = stageStartTime.minusWeeks(1);
```

Comment:
```suggestion
            LocalDateTime currentTime = stageStartTime.minusWeeks(1);
```
currentTime이라는 네이밍은 어떤가요?